src/pages/markets/static/content/_cryptocurrencies.tsx

```diff
@@ -10,26 +10,20 @@ export type CryptocurrenciesContent = {
     text: React.ReactNode
 }
 
```

Comment:
Please remove CryptocurrenciesContent type inside this file because we don't use it anywhere